system-reinstall-bootc/src/main.rs

```diff
@@ -10,6 +10,7 @@ mod prompt;
 pub(crate) mod users;
 
 const ROOT_KEY_MOUNT_POINT: &str = "/bootc_authorized_ssh_keys/root";
+const SSH_KEY_TEMP_FILE: &str = "/tmp/.system-reinstall-bootc-ssh-keys";
```

Comment:
[Predicable filenames in /tmp](https://cwe.mitre.org/data/definitions/377.html) can lead to security issues. Do we really need a hardcoded path here? Can't we just allocate a `tempfile::NamedTempFile()`? 

If we do need a hardcoded path, one mitigation is to use `/run/bootc-ssh-keys` or so, as that's a directory only root should be able to write to.